activity.js

```diff
@@ -8,7 +8,23 @@ class Activity {
     this.id = id;
   };
   countdown(){
-    console.log();
+    var time = Number(this.minutes * 60) + Number(this.seconds)
+    console.log(time);
+    var decreaseTime = setInterval(function() {
+        var minutes = String(Math.trunc(time / 60)).padStart(2, 0)
+        var seconds = String(Math.trunc(time % 60)).padStart(2, 0)
+     //display complete on the buttonTitles
+     timer.textContent = `${minutes}:${seconds}`;
+
+      //decrease time
+        time--;
+      //clear timer
+
+
+
+      //log activity button displays
```

Comment:
Should we keep the comments in?

Reply:
This method is unfinished currently, so I think it's ok to keep the pseudocode in there